config/scenario_config.csv

```diff
@@ -1,54 +1,54 @@
 # H12 SSP2EU;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
-SSP2EU-calibrate;0;calibrate;14;;;;;;0;;;2100;;;off;;;;;;;;;;;;;;;;EUR_regi, NEU_regi;;;;;;;;;;;;;;;;heat;Mix1;2005;;;;;SSP2EU-calibration: This baseline scenario follows the Shared Socioeconomic Pathways 2 called Middle of the Road.
-SSP2EU-NPi-calibrate;0;calibrate;14;;;rcp45;;;0;NPi_Baseline;1;2100;9;;;;;;;;;;;NPi2018;regiCarbonPrice;2030.EUR_regi.tax.t.FE_wo_b_wo_n_e.all 1.2809;;2020.2030.EUR_regi.all.year.netGHG_LULUCFGrassi 3.266;on;on;EUR_regi, NEU_regi;;;;;;;;;;3;;;;;;heat;Mix1;2005;;;;;SSP2EU-NPi-calibrate: This reference policy/baseline scenario follows the Shared Socioeconomic Pathways 2 called Middle of the Road. 
-SSP2EU-Base;1;;;;;;;;0;;;2100;;;off;;;;;;;;;;;;;;;;EUR_regi, NEU_regi;;;;;;;;;;;;;;;;heat;Mix1;2005;;;;;SSP2EU-Base: This baseline calibration scenario follows the Shared Socioeconomic Pathways 2 called Middle of the Road.
-SSP2EU-NDC;1;;;;;rcp45;3;globallyOptimal;0;NDC;1;2100;9;;;;;;;;;;;NDC;regiCarbonPrice;2030.EUR_regi.tax.t.FE_wo_b_wo_n_e.all 1.2809;;2020.2030.EUR_regi.all.year.netGHG_LULUCFGrassi 2.450;on;on;EUR_regi, NEU_regi;;;;;;;;;;3;;;;;;heat;Mix2;2015;;SSP2EU-Base;SSP2EU-Base;;SSP2EU-NDC: This Nationally Determined Contribution (NDC) scenario follows the Shared Socioeconomic Pathways 2 called Middle of the Road. The NDC includes all pledged policies even if not yet implemented. It assumes that the moderate and heterogeneous climate ambition reflected in the NDCs at the begining of 2021 continues over the 21st century. This scenario serves as reference for all other policy scenarios and The NPi scenario until 2020.
+SSP2EU-calibrate;calibrate,AMT;calibrate;14;;;;;;0;;;2100;;;off;;;;;;;;;;;;;;;;EUR_regi, NEU_regi;;;;;;;;;;;;;;;;heat;Mix1;2005;;;;;SSP2EU-calibration: This baseline scenario follows the Shared Socioeconomic Pathways 2 called Middle of the Road.
+SSP2EU-NPi-calibrate;calibrate;calibrate;14;;;rcp45;;;0;NPi_Baseline;1;2100;9;;;;;;;;;;;NPi2018;regiCarbonPrice;2030.EUR_regi.tax.t.FE_wo_b_wo_n_e.all 1.2809;;2020.2030.EUR_regi.all.year.netGHG_LULUCFGrassi 3.266;on;on;EUR_regi, NEU_regi;;;;;;;;;;3;;;;;;heat;Mix1;2005;;;;;SSP2EU-NPi-calibrate: This reference policy/baseline scenario follows the Shared Socioeconomic Pathways 2 called Middle of the Road.
```

Comment:
should be 0 for now, currently placeholder for the NPi calibration, but need to check settings again